src/signals/incident/definitions/wizard-step-2-vulaan.js

```diff
@@ -45,6 +45,12 @@ const expandQuestions = memoize(
         }),
         {}
       ),
+      error: {
+        meta: {
+          label: 'Nog niet alles is correct ingevuld. Volg de meldingen in het rood. Scroll eventueel weer naar boven.',
```

Comment:
To skip this step when no additional questions are required:

```suggestion
          label: 'Nog niet alles is correct ingevuld. Volg de meldingen in het rood. Scroll eventueel weer naar boven.',
          ignoreVisibility: true,
```